From 3d3285d6addca6ac554e3e036815015c836d477d Mon Sep 17 00:00:00 2001
From: Alfonso Ruelas <alfonso@ruelas.org>
Date: Wed, 6 Oct 2021 22:32:33 -0700
Subject: [PATCH] Fixed bug with default sorting method

---
 Files/Helpers/ItemListDisplayHelpers/SortingHelper.cs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Files/Helpers/ItemListDisplayHelpers/SortingHelper.cs b/Files/Helpers/ItemListDisplayHelpers/SortingHelper.cs
index a5a7d3c0..65ce63e5 100644
--- a/Files/Helpers/ItemListDisplayHelpers/SortingHelper.cs
+++ b/Files/Helpers/ItemListDisplayHelpers/SortingHelper.cs
@@ -25,7 +25,8 @@ namespace Files.Helpers
                 SortOption.FileTag => item => item.FileTag,
                 SortOption.OriginalFolder => item => (item as RecycleBinItem)?.ItemOriginalFolder,
                 SortOption.DateDeleted => item => (item as RecycleBinItem)?.ItemDateDeletedReal,
-                _ => null,
+                //_ => null,
+                _ => item => item.ItemName,
             };
         }
 
-- 
2.31.1.windows.1

